drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -0,0 +1,60 @@
+:scss
+  .user-permission-report td ul {
+    list-style: none;
+    padding-left: 0;
+  }
+  li {
+    margin-bottom: 0.5em;
+  }
+:ruby
+  title = _('User Permission Report')
+  content_for :title, title
```

Comment:
dropping into a ruby block feels a bit excessive for just two lines.  Also, `title` isn't called again, so I'd suggest `= content_for :title, _('User Permission Report')` unless I'm missing something

Reply:
✅